pymeasure/instruments/common_base.py

```
@@ -254,18 +254,18 @@ def get_channels(cls):
                 channels.append((name, member))
         return channels
 
-    @staticmethod
+    @classmethod
     def get_channel_pairs(cls):
         """Return a list of all the Instrument's channel pairs"""
         channel_pairs = []
-        for name, creator in CommonBase.get_channels(cls):
+        for name, creator in cls.get_channels():
```

Ruff (0.15.12)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)